src/hooks/c-bridge/useCbridgeV2.ts

```diff
@@ -63,6 +70,10 @@ export function useCbridgeV2() {
         })
     )) as SelectedToken[];
 
+    tokens.value = tokens.value.filter((token) => {
+      return token !== null;
```

Comment:
this `null` check and `return null` above seem dangerous to me since the value can also be undefined. Any way to improve this?

Reply:
Is [this](https://github.com/AstarNetwork/astar-apps/pull/291/commits/e3a2701e6f3d5350a0ee78449c7f993fc74649ba) commit make sense to you?